.github/workflows/latest_dependency_checker.yml

```diff
@@ -31,4 +31,4 @@ jobs:
         branch: latest-dep-update
         branch-suffix: short-commit-hash
         base: main
-        team-reviewers: @alteryx/ml-tools
+        team-reviewers: alteryx/ml-tools
```

Comment:
based on how team-reviewers is used in [this example](https://github.com/peter-evans/create-pull-request#reference-example), do you think just `ml-tools` would work?

Reply:
Fixed. I also lowered the pandas version so we can test this.